### PR TITLE
refactor Statistics

### PR DESCRIPTION
**Pull request by Permas-Teo**
implement StatisticsManager to update StatisticsDisplay

### CODE REVIEW

File: src/main/java/seedu/address/logic/StatisticsManager.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    public void updateStatistics(ObservableList<DayData> customQueue) {
+        this.customQueue = customQueue;
+        DayData latestDayData = customQueue.get(CONSTANT_SIZE - 1);
+        int currProgress = latestDayData.getPomDurationData().value;
+
+        String currTargetText = statistics.getDailyTarget();
+        int currTarget = Integer.valueOf(currTargetText);
+        if (currProgress >= currTarget) {
```
Comment (BransonNg) [inline src/main/java/seedu/address/logic/StatisticsManager.java:29]: actually should we provide feedback that they've maxed it out

Comment (BransonNg) [reply src/main/java/seedu/address/logic/StatisticsManager.java:29]: hahah maybe can set bar with star for 1 second and set it back to bar without star.
Star meaning they maxed it out already

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -336,7 +341,8 @@ private CommandResult executeCommand(String commandText)
                         .select(switchTabCommandResult.getTabToSwitchIndex());
                 if (switchTabCommandResult.getTabToSwitchIndex() == STATS_TAB_INDEX) {
                     ObservableList<DayData> customQueue = logic.getCustomQueue();
-                    statisticsDisplay.updateGraphs(customQueue);
+                    statisticsManager.updateStatistics(customQueue);
+                    this.updateStatisticsDisplay();
```
Comment (BransonNg) [inline src/main/java/seedu/address/ui/MainWindow.java:345]: we can't call this within statisticsDisplay itself? 
[just thinking of ways to reduce the crowding in MainWindow haha]

**Comment by BransonNg** [comment]
niceu, very OOP. lgtm!

**Review by BransonNg — APPROVED** [review]

File: src/main/java/seedu/address/logic/StatisticsManager.java (hunk 1 above)
Comment (Permas-Teo) [reply src/main/java/seedu/address/logic/StatisticsManager.java:29]: sounds good! let's address it in another issue

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (Permas-Teo) [reply src/main/java/seedu/address/ui/MainWindow.java:345]: not with current implementation as mainwindow ties manager to display!

**Pull request merged**